cranelift/codegen/src/egraph.rs

```diff
@@ -483,6 +534,9 @@ impl<'a> EgraphPass<'a> {
         let mut effectful_gvn_map: ScopedHashMap<(Type, InstructionData), Value> =
             ScopedHashMap::new();
 
+        let mut available_block: SecondaryMap<Value, Block> =
+            SecondaryMap::with_default(Block::reserved_value());
```

Comment:
Might be worth pre-reserving space here, since IIUC we are going to end up filling this in for every single value.